.travis.yml

```diff
@@ -0,0 +1,46 @@
+# language target
+language: cpp
+compiler: gcc
+
+# main build matrix
+matrix:
+  include:
+    - name: "Bionic Ubuntu Linux"
+      os: linux
+      dist: bionic
+      sudo: required
+
+
+install:
+  - sudo apt update -y
+  # Instal build dependencies
+  - sudo apt install -y build-essential g++ cmake libarchive-dev extra-cmake-modules qtbase5-dev libqt5webkit5-dev libqt5x11extras5-dev libxcb-keysyms1-dev  libsqlite3-dev qtbase5-private-dev xmlto libqt5webkit5-dev libappindicator-dev
+
+  # Install appimage-builder
+  - sudo apt install -y build-essential python3 python3-setuptools python3-pip patchelf gnupg2 file desktop-file-utils fakeroot
+  - sudo pip3 install git+https://github.com/AppImageCrafters/appimage-builder.git
+
+  # Install appimagetool
+  - wget https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage -O /tmp/appimagetool
```

Comment:
Is it guaranteed to be always compatible, and new version will not suddenly stop working? Just wondering if it's fine to use unversioned download.

Reply:
The AppImage team hasn't changed the command interface in years. But as a continuous release, there is always a small chance for a breaking change. As you suggest it will be better to use the latest release available.